templates/interface/Debian.erb

```diff
@@ -9,7 +9,7 @@ allow-hotplug <%= @interface %>
 <% if @manage_address -%>
     address <%= @manage_address %>
 <% end -%>
-<% if @manage_hwaddr -%>
+<% if @manage_address and ! @manage_address.empty? -%>
```

Comment:
With dhcp interfaces that line should not be there. And now it's set to '' (empty string) which evaluates to true resulting in lines with just the word address in stanza's that don't need that line.